report/main.tex

```diff
@@ -52,18 +63,171 @@ \section{Introduction}
+
+The prompt for ``rolling summary'' replaces the key names (``regular-chaotic-motions'', etc.) with ``current-summary'', and the stored summaries summarize both the current chunk and all previous chunks. The final example shows the model reading just the single key ``current-summary'' and combining it with the input chunk to produce the answer. Verbiage in the explanations is also slightly adapted to match these differences.
+
+
+\section{Results}
+
+We report metrics from the ROUGE family~\cite{lin-hovy-2003-automatic,ganesan2015rouge} for the three approaches on \texttt{n=10} samples from the validation set.
 
+\begin{table}[!ht]
+  \centering
+  \begin{tabular}{c|c|c|c|c}
+    \textbf{Method} & \textbf{ROUGE} & \textbf{ROUGE-2} & \textbf{ROUGE-L} & \textbf{ROUGE-L-Sum} \\
+    \hline
+    base            & 11.7           & 4.0              & 6.1              & 8.2                  \\
+    compressed      & 29.4           & 9.7              & 15.1             & 23.1                 \\
+    rolling         & 34.2           & 8.8              & 18.7             & 27.3                 \\
+  \end{tabular}
+  \vspace{0.5cm}
+  \caption{Experimental results for each of the prompts described in Section~\ref{section:experiments} on 10 samples from the arXiv summarization dataset, evaluated on several ROUGE-based metrics.}\label{tab:results}
+\end{table}
 
-\section{Supplementary Material}
+We observe clear improvement over the base prompt for the two modifications. While the sample size for this experiment was small,\footnote{The cost of this experiment with \texttt{n=10} was US\$16, so we would like more funds in order to validate that these changes improve performance.} we think the difference is significant because they make it possible for more stored information to be \texttt{[read]} when generating the final summary.
 
-Extra stuff to read, if you want.
 
 
+\section{Conclusion}
+
+- context window is limiting to ART framework
+- future work in fine tuning a model instead of using a frozen one.
+- lightweight models may be more appropriate
+- Could be used for better conversational AI
+
+In this project we have designed and implemented tools to be used by an LLM to organize and store relevant data for long-term/future use.
+We encountered difficulties in providing enough context for the LLM to properly use the tools, but found some success with simplified run strategies.
+Future research avenues could involve using newer, bigger, LLMs that have larger context windows allowing for more varied tool use samples to be passed as part of the prompt.
+Otherwise, it may be desirable to fine tune smaller models specifically to use tools and address long standing issues with LLM capabilities.
+To achieve the goal of an AGI system, it will be necessary for it to diversify its problem-solving across many contexts.
+Creating systems capable of learning to use tools may be an invaluable step along this path.
```

Comment:
```suggestion
Creating systems capable of learning to use tools will be an invaluable step in the path to robust models that learn across their deployment lifecycle.
```

Sorry, I don't want to directly attach our project to the idea of AGI, even though that's the name of the class :P